.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+    - id: black
+      language_version: python3.8
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
Is there a reason not to use https://github.com/pycqa/flake8 as the repo?

Reply:
You're right, no good reason to use gitlab. Changed it.